features/step_definitions/aspects_steps.rb

```diff
@@ -88,6 +88,21 @@ def aspect_dropdown_visible?
   end
 end
 
+When /^I drag "([^"]*)" (up|down) (\d+) pixels?$/ do |aspect_name, direction, distance|
+  distance = distance.to_i * -1 if direction == 'up'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.